.github/comment-commands/pending.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#doc: Add a REQUESTED_CHANGE type review to mark issue non-mergeable: `/pending <reason>`
+# shellcheck disable=SC2124
+MESSAGE="Marking this issue as un-mergeable as requested.
+
+Please use \`/ready\` comment when it's resolved.
+
+> $@"
+
+URL="$(jq -r '.issue.pull_request.url' "$GITHUB_EVENT_PATH")/reviews"
+set +x #GITHUB_TOKEN
```

Comment:
Is `set +x` needed for safety in the other scripts, or is this one special?

Reply:
I am not sure if github expose the token to the console or it's masked. If not masked we should either use `set +x` or never use `set -x`. Both works. Do you prefer to put it to everywhere or remove it from everwhere?

Reply:
I wouldn't have thought about it, but since you added it here, I guess I'd prefer adding it everywhere.

Reply:
Ok. I added `set +x` to everywhere where GITHUB_TOKEN is used.